docs/config.md

```diff
@@ -62,16 +62,16 @@ for more details.
-| `OTEL_DOTNET_AUTO_LOGS_INSTRUMENTATION_ENABLED`        | Enables all log instrumentations. Overrides `OTEL_DOTNET_AUTO_INSTRUMENTATION_ENABLED`.                                                                                                                                                                         | Same as `OTEL_DOTNET_AUTO_INSTRUMENTATION_ENABLED`         |
-| `OTEL_DOTNET_AUTO_LOGS_{0}_INSTRUMENTATION_ENABLED`    | Configuration pattern for enabling or disabling a specific log instrumentation, where `{0}` is the case-sensitive name of the instrumentation you want to enable. Overrides `OTEL_DOTNET_AUTO_LOGS_INSTRUMENTATION_ENABLED`.                                    | Same as `OTEL_DOTNET_AUTO_LOGS_INSTRUMENTATION_ENABLED`    |
+| Environment variable                                    | Description                                                                                                                                                                                                                                                     | Default value                                               |
+|---------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------|
+| `OTEL_DOTNET_AUTO_INTEGRATIONS_FILE`                    | List of bytecode instrumentations JSON configuration filepaths, delimited by the platform-specific path separator (`;` on Windows, `:` on Linux and macOS). For example: `%ProfilerDirectory%/integrations.json`. It is required for bytecode instrumentations. |                                                             |
+| `OTEL_DOTNET_AUTO_INSTRUMENTATION_DISABLED`             | Disables all instrumentations.                                                                                                                                                                                                                                  | `false`                                                     |
+| `OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_DISABLED`      | Disables all trace instrumentations. Overrides `OTEL_DOTNET_AUTO_INSTRUMENTATION_DISABLED`.                                                                                                                                                                     | Same as `OTEL_DOTNET_AUTO_INSTRUMENTATION_DISABLED`         |
```

Comment:
maybe?

```suggestion
| `OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_DISABLED`      | Disables all trace instrumentations. Overrides `OTEL_DOTNET_AUTO_INSTRUMENTATION_DISABLED`.                                                                                                                                                                     | Derived from `OTEL_DOTNET_AUTO_INSTRUMENTATION_DISABLED`         |
```

Reply:
Suggested some changes to Piotr

Reply:
d4398104102607006798baa30f67b8d22d612dba